src/backend/TrafficCourts/Citizen.Service/Models/Tickets/ViolationTicket.cs

```diff
@@ -0,0 +1,258 @@
+    /// <summary>
+    /// The surname or corporate name.
+    /// </summary>
+    [JsonPropertyName("surname")]
+    [MaxLength(30)]
+    public string? Surname { get; set; }
+
+    /// <summary>
+    /// The given names or corporate name continued.
+    /// </summary>
+    [JsonPropertyName("given_names")]
+    [MaxLength(30)]
+    public string? GivenNames { get; set; }
+
+    /// <summary>
+    /// The person issued the ticket has been identified as a young person.
+    /// </summary>
+    [JsonPropertyName("is_young_person")]
+    public bool? IsYoungPerson { get; set; }
+
+    #region Drivers Licence
+
+    /// <summary>
+    /// The drivers licence number. Note not all jurisdictions will use numeric drivers licence numbers.
+    /// </summary>
+    [JsonPropertyName("drivers_licence_number")]
+    [MaxLength(20)]
+    public string? DriversLicenceNumber { get; set; }
+
+    /// <summary>
+    /// The province or state the drivers licence was issued by.
+    /// </summary>
+    [JsonPropertyName("drivers_licence_province")]
+    [MaxLength(30)]
+    public string? DriversLicenceProvince { get; set; }
+
+    /// <summary>
+    /// The year the drivers licence was produced.
+    /// </summary>
+    [JsonPropertyName("drivers_licence_produced_year")]
+    [Range(1900, 2100)]
+    public short? DriversLicenceProducedYear { get; set; }
+
+    /// <summary>
+    /// The year the drivers licence expires.
+    /// </summary>
+    [JsonPropertyName("drivers_licence_expiry_year")]
+    [Range(1900, 2100)]
+    public short? DriversLicenceExpiryYear { get; set; }
+    #endregion
+
+    /// <summary>
+    /// The birthdate of the individual the violation ticket was issued to.
+    /// </summary>
+    [JsonPropertyName("birthdate")]
+    [SwaggerSchema(Format = "date")]
+    public DateTime? Birthdate { get; set; }
+
+    /// <summary>
+    /// The address of the individual the violation ticket was issued to.
+    /// </summary>
+    [JsonPropertyName("address")]
+    [MaxLength(30)]
+    public string? Address { get; set; }
+
+    /// <summary>
+    /// The city of the individual the violation ticket was issued to.
+    /// </summary>
+    [JsonPropertyName("city")]
+    [MaxLength(30)]
+    public string? City { get; set; }
+
+    /// <summary>
+    /// The province or state of the individual the violation ticket was issued to.
+    /// </summary>
+    [JsonPropertyName("province")]
+    [MaxLength(30)]
+    public string? Province { get; set; }
+
+    /// <summary>
+    /// The postal code or zip code.
+    /// </summary>
+    [JsonPropertyName("postal_code")]
+    [MaxLength(6)]
+    public string? PostalCode { get; set; }
+
+    /// <summary>
+    /// The address represents a change of address. The address on the violation would be different from the address 
+    /// on the drivers licence or provided identification.
+    /// </summary>
+    [JsonPropertyName("is_change_of_address")]
+    public bool? IsChangeOfAddress { get; set; }
```

Comment:
Should this and other `bool `properties be nullable?  Either the property on the form is checked or not, but a `bool?` is a trinary object (true, false, null).